OpenRA.Mods.RA2/Activities/EnterGarrison.cs

```diff
@@ -0,0 +1,220 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)

```

Comment:
The year is wrong. There should also be a normalized header limited for this project's scope. It's not necessarily OpenRA authors afterall.

Reply:
That can be left to a followup though.

Reply:
I have been using OpenRA's header as is everywhere. So it would be better done in another PR indeed.